src/main/java/core/basesyntax/strategy/BalanceTransaction.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.DatabaseImpl;
+import core.basesyntax.models.Transaction;
+
+public class BalanceTransaction implements TransactionHandler {
+    private DatabaseImpl storage = new DatabaseImpl();
```

Comment:
це варто ініціалізувати через конструктор
```suggestion
    private Database storage;
```
те ж саме стосується інших хендлерів